sdk/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "risc0-ethereum-sdk"
```

Comment:
A good part of this SDK (the prover.rs / snark.rs) seems really useful to maybe move into the bonsai-sdk inside of risc0? Maybe as a follow up later on for 0.21?

Reply:
Yeah I'm actually of the idea the this sdk should be moved out from this repo. What would be ideal is to have a `risc0-ethereum` or `risc0-web3` repo to have all of this sdk together with the ethereum contracts that now live on `risc0`. 
Als, we have nice ideas for this sdk that we'll start implementing soon and so it does not really fit in here